File: app/src/main/java/app/android/ttp/mikazuki/yoshinani/domain/entity/Total.java
package app.android.ttp.mikazuki.yoshinani.domain.entity;

/**
 * Created by haijimakazuki on 15/08/16.
 */
public class Total {

    private double paid;
    private double toPay;
    private int groupId;
    private int userId;

    public Total() {
    }

    public Total(long paid, long toPay, int groupId, int userId) {
        this.paid = paid;
        this.toPay = toPay;
        this.groupId = groupId;
        this.userId = userId;
    }

    public double getPaid() {
        return paid;
    }

    public void setPaid(double paid) {
        this.paid = paid;
    }

    public double getToPay() {
        return toPay;
    }

    public void setToPay(double toPay) {
        this.toPay = toPay;
    }

    public int getGroupId() {
        return groupId;
    }

    public void setGroupId(int groupId) {
        this.groupId = groupId;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }
}

File: app/src/main/java/app/android/ttp/mikazuki/yoshinani/repository/retrofit/entity/Total.kt
package app.android.ttp.mikazuki.yoshinani.repository.retrofit.entity

/**
 * Created by haijimakazuki on 15/08/16.
 */
class Total {

    var paid: Double = 0.toDouble()
    var toPay: Double = 0.toDouble()
    var groupId: Int = 0
    var userId: Int = 0

    constructor() {}

    constructor(paid: Long, toPay: Long, groupId: Int, userId: Int) {
        this.paid = paid.toDouble()
        this.toPay = toPay.toDouble()
        this.groupId = groupId
        this.userId = userId
    }
}
